app/views/includes/scripts.njk

```diff
@@ -57,3 +57,19 @@
   });
 </script>
 <script src="{{ js_path }}"></script>
+<script>
+{# If our JS bundle fails to load we need to undo Apple Pay check #}
+function ready(fn) {
+  if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading"){
+    fn();
+  } else {
+    document.addEventListener('DOMContentLoaded', fn);
+  }
+}
+
+ready(() => {
```

Comment:
Are we OK to use arrow functions? Presumably JavaScript inside Nunjucks templates doesn’t go through the compiler as JavaScript in external files would.

Reply:
OOF! Good point